benches/mantis/jobs/mantis.ncl

```diff
@@ -1,260 +1,316 @@
 let types = import "../schemas/nomad/types.ncl" in
-let promtail = import "../tasks/promtail.ncl" in
-let Telegraf = import "../tasks/telegraf.ncl" in
-let lib = import "../lib.ncl" in
+  let promtail = import "../tasks/promtail.ncl" in
+    let Telegraf = import "../tasks/telegraf.ncl" in
+      let lib = import "../lib.ncl" in
```

Comment:
This might end up being a bit of a pain point. Nickel doesn't allow multiple bindings in a single let (yet), so many files will start with a number of `let` statements. While it makes sense for a let to increase the indentation elsewehere, maybe at top level that shouldn't be the case? I have no idea if that's possible with topiary at the moment.

Reply:
I think OCaml and SML don't increase indentation with a let. It's really similar to a variable binding in other languages, and that doesn't increase indentation either. I vouch for never indenting further the body of a let (you have the same style of block of let-definitions at the level of functions as well, and in fact potentially pretty much anywhere, arbitrarily deep)

Reply:
Removed the indentation after `let...in...` in https://github.com/tweag/topiary/commit/a26a8708eb5524abbf36468594cfb85a66633429. I'll run Topiary against the Nickel files again once I've figured out [this problem](https://github.com/tweag/nickel/pull/1198#discussion_r1147862341).